pages/coinbase/signin.py

```diff
@@ -49,20 +49,19 @@ def __init__(self, driver, spawn=False, bypass_auth=True):
 
     # BODY
 
-    email_field_loc =               (by.ID, 'email')
-    pass_field_loc =                (by.ID, 'password')
-    stay_signed_in_checkbox_loc =   (by.ID, 'stay_signed_in')
-    sign_in_button_loc =            (by.ID, 'signin_button')
-    forgot_password_link_loc =      (by.LINK_TEXT, 'Forgot password?')
-    no_account_link_loc =           (by.LINK_TEXT, "Don't have an account?")
-    privacy_policy_link_loc =       (by.PARTIAL_LINK_TEXT, 'Privacy Policy')
+    email_field_loc =                  (by.ID, 'email')
+    pass_field_loc =                   (by.ID, 'password')
+    keep_me_signed_in_checkbox_loc =   (by.ID, 'stay_signed_in')
+    sign_in_button_loc =               (by.ID, 'signin_button')
+    forgot_password_link_loc =         (by.LINK_TEXT, 'Forgot password?')
+    not_have_account_link_loc =        (by.LINK_TEXT, "Don't have an account?")
+    privacy_policy_link_loc =          (by.PARTIAL_LINK_TEXT, 'Privacy Policy')
     two_factor_link_loc = (
         by.LINK_TEXT, 'Have an issue with 2-factor authentication?')
 
     # MISC
 
-    #sign_in_error_alert_loc = (by.LINK_TEXT, 'Invalid email or password.')
-    sign_in_error_alert_loc = (by.CLASS_NAME, 'alert')
+    sign_in_error_alert_loc = (by.LINK_TEXT, 'Invalid email or password.')
```

Comment:
why did you change this?

Reply:
the commented out line didn't belong there, but then I decided it was a better locator than the class name locator.  But upon thinking about it further class name is better than link text so I'll change it back